google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineEnvironment.java

```diff
@@ -23,7 +23,8 @@
  */
 public enum AppEngineEnvironment {
   APP_ENGINE_STANDARD("appengine.environment.name.standard"),
-  APP_ENGINE_FLEX("appengine.environment.name.flexible");
+  APP_ENGINE_FLEX("appengine.environment.name.flexible"),
+  APP_ENGINE_FLEX_COMPAT("appengine.environment.name.flexible");
```

Comment:
I feel like this topic is almost deserving of a design doc :rofl: 

Another issue I see with renaming to something else, is that we'd possibly need two object types instead of one, for the project type and for the environment.

We could arguably also start considering AppEngineEnvironment as an internal environment, rather than remote environment?